src/components/Navbar/Navbar.tsx

```diff
@@ -72,19 +83,43 @@ const Navbar = () => {
                         horizontal: 'right'
                     }}
                 >
-                    <MenuItem onClick={handleCLose}>
-                        <Link to={'/'}>
-                            <Button color='inherit'>Current Location</Button>
+                    <MenuItem
```

Comment:
make an object and use map  you wrote the same code 3 times

Reply:
done !